isb_web/config.py

```diff
@@ -39,7 +39,7 @@ class Settings(BaseSettings):
     # This shouldn't be checked in.  Set by doing export DATACITE_PASSWORD=foobar
     datacite_password: str = ""
 
-    orcid_hostname: str = "pub.sandbox.orcid.org"
+    orcid_hostname: str = "orcid.org"
```

Comment:
This is only used for local development, but there's no reason to be using the sandbox anymore.